README.md

```diff
@@ -116,14 +116,24 @@ The queues are specified in a custom configuration file. For example, create a `
 ````
 include classpath("application.conf")
 
-queues {
-    queue1 {
+queues = [
```

Comment:
If possible, I would prefer not to change the format of the `queues` config, so that people already using this won't have to migrate.

Some note in the readme about DLQ support and how it works would be great as well.